scripts/output_code_size.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
Nope.  This should either be:

```
#!/bin/sh
```

```
#!/usr/bin/env bash
```

And we should not be using bash extensions in portable scripts, so it should be the former.

Reply:
My bad! This is leftover from a previous version of the script...